resources/route53_health_checks.go

```diff
@@ -0,0 +1,237 @@
+				Type:     schema.TypeInt,
+				Resolver: schema.PathResolver("HealthCheckConfig.RequestInterval"),
+			},
+			{
+				Name:     "resource_path",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("HealthCheckConfig.ResourcePath"),
+			},
+			{
+				Name:     "search_string",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("HealthCheckConfig.SearchString"),
+			},
+			{
+				Name: "health_check_version",
+				Type: schema.TypeBigInt,
+			},
+			{
+				Name:     "resource_id",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("Id"),
+			},
+			{
+				Name:     "cloud_watch_alarm_configuration_comparison_operator",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("CloudWatchAlarmConfiguration.ComparisonOperator"),
+			},
+			{
+				Name:     "cloud_watch_alarm_configuration_evaluation_periods",
+				Type:     schema.TypeInt,
+				Resolver: schema.PathResolver("CloudWatchAlarmConfiguration.EvaluationPeriods"),
+			},
+			{
+				Name:     "cloud_watch_alarm_configuration_metric_name",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("CloudWatchAlarmConfiguration.MetricName"),
+			},
+			{
+				Name:     "cloud_watch_alarm_configuration_namespace",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("CloudWatchAlarmConfiguration.Namespace"),
+			},
+			{
+				Name:     "cloud_watch_alarm_configuration_period",
+				Type:     schema.TypeInt,
+				Resolver: schema.PathResolver("CloudWatchAlarmConfiguration.Period"),
+			},
+			{
+				Name:     "cloud_watch_alarm_configuration_statistic",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("CloudWatchAlarmConfiguration.Statistic"),
+			},
+			{
+				Name:     "cloud_watch_alarm_configuration_threshold",
+				Type:     schema.TypeFloat,
+				Resolver: schema.PathResolver("CloudWatchAlarmConfiguration.Threshold"),
+			},
+			{
+				Name:     "linked_service_description",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("LinkedService.Description"),
+			},
+			{
+				Name:     "linked_service_service_principal",
+				Type:     schema.TypeString,
+				Resolver: schema.PathResolver("LinkedService.ServicePrincipal"),
+			},
+		},
+	}
+}
+
+// ====================================================================================================================
+//                                               Table Resolver Functions
+// ====================================================================================================================
+func fetchRoute53HealthChecks(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan interface{}) error {
+	var config route53.ListHealthChecksInput
+	c := meta.(*client.Client)
+	svc := c.Services().Route53
+
+	for {
+		response, err := svc.ListHealthChecks(ctx, &config, func(o *route53.Options) {})
+		if err != nil {
+			return err
+		}
+		res <- response.HealthChecks
```

Comment:
In theory for every 10 HealthChecks you can do one request with ListTagsForResources. In that case you need to wrap the HealthChecks structs with a custom health check struct:

`struct HealthCheck {
route53.HealtchCheck
tags map[string]interface{}
}`

And build them in fetchRoute53HealthChecks, the sdk should take care of the rest. 

The same is to be said on HostedZones